brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Spacing control - Added `%` unit support.

### Description

- Spacing control - Added % unit support.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create section block in master add padding/margin in `%`.
- Switch to next-release then check padding/margin option `%` is missing there.
- Switch to `section-margin-padding-unit` % option is visible now.

### Checklist:
